src/transform/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
     "target": "ES2022",
-    "module": "ES2020",
-    "moduleResolution": "Node",
+    "module": "es2022",
+    "moduleResolution": "Bundler",
```

Comment:
I'm curious about these changes.  It seems to work, but I recall fussing with them awhile ago and it didn't then.  Did this actually change anything in the output js?